docs/Language/HTML/HTML.md

```diff
@@ -6,11 +6,72 @@ parent: Language
 has_children: false
 ---
 
-{{ page.title }}
-======================
+# {{ page.title }}
 
-{% include under_construction.html %}
+## What is HTML?
```

Comment:
I would consider removing the following ` ## What is HTML?  ` as the ` # {{ page.title }} ` would let the user know what it is about "HTML " and the describtion written states what the following is.